docs/src/integrations/iceberg.md

```diff
@@ -412,6 +409,181 @@ sequenceDiagram
     - [Iceberg Official Documentation](https://iceberg.apache.org/docs/latest/)
     - [lakeFS Enterprise Features](../enterprise/index.md)
 
+## Catalog Sync
+
+Catalog Sync allows you to synchronize Iceberg tables between the lakeFS catalog and external catalogs such as AWS Glue Data Catalog or other Iceberg REST-compatible catalogs.
+This enables workflows where some users or tools need to access data through other external catalogs.
+
+### Use Cases
+
+**Collaboration with External Tools**: Share data with users who rely on tools that only support specific catalogs.
+For example, data engineers can work with tables in lakeFS while data analysts query the same data through AWS Glue using Athena, all while maintaining isolation and version control.
+
+```mermaid
+sequenceDiagram
+    actor Alice (Data Engineer)
```

Comment:
The actor names must match the ones used below exactly - for me this renders wrong:

<img width="1010" height="396" alt="Image" src="https://github.com/user-attachments/assets/a0beafb4-b1dd-4db0-989c-4072dbebf377" />